ui/component/publish/livestream/publishLivestream/index.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
```

Comment:
- [ ] Replays should be tied to the channel (unless it's a new behavior that we want).

Steps:
1. Start and end a livestream.
2. Try to publish the replay
3. In the final step, change the channel in the channel selector.
    - The old GUI would update the replay selection list based on the selected channel.

Reply:
The replay area should already be tied to the channel and it seems to work fine here. Did you trigger some kind of edge case?

Reply:
Oh, I entered from "Update or publish replay".  Also happening in production.

> _unless it's a new behavior that we want_

Maybe it was intentional, to allow publishing to another channel ... not sure.

Reply:
Can leave as-is, I guess.